Gems/EMotionFX/Code/EMotionFX/Source/Actor.cpp

```diff
@@ -1368,11 +1368,14 @@ namespace EMotionFX
     // update the static AABB (very heavy as it has to create an actor instance, update mesh deformers, calculate the mesh based bounds etc)
     void Actor::UpdateStaticAabb()
     {
-        ActorInstance* actorInstance = ActorInstance::Create(this, nullptr, m_threadIndex);
-        actorInstance->UpdateMeshDeformers(0.0f);
-        actorInstance->UpdateStaticBasedAabbDimensions();
-        actorInstance->GetStaticBasedAabb(&m_staticAabb);
-        actorInstance->Destroy();
```

Comment:
Yes, it is. It was creating an instance, just to skin it in the bind pose, just to get an aabb, which is equivalent to the aabb of the static mesh.

Doing that spawned jobs, and it would be kicked off from inside an asset handling job, and in some cases the pool of job threads was starved, with all the threads waiting for something on the main thread, but the main thread waiting for an asset system lock being held by this thread, and this thread unable to finish skinning because there are no workers available to do the work. I'll add that to the PR description and commit message. I wanted to add the description there in the first place, but couldn't remember what the actual deadlock looked like :)